openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -1021,7 +1021,7 @@ If the Wallet is requesting the issuance of a Credential that is not supported b
   * `invalid_credential_request`: The Credential Request is missing a required parameter, includes an unsupported parameter or parameter value, repeats the same parameter, or is otherwise malformed.
   * `unsupported_credential_type`: Requested Credential type is not supported.
   * `unsupported_credential_format`: Requested Credential Format is not supported.
-  * `invalid_proof`: The `proof` or `proofs` parameter in the Credential Request is invalid. For example, because either both fields are missing or both are present simultaneously, or one of the provided key proofs is either invalid or not linked to a nonce provided by the Credential Issuer.
+  * `invalid_proof`: The `proof` or `proofs` parameter in the Credential Request is invalid: (1) if either both fields are missing, (2) both are present simultaneously, (3) one of the provided key proofs is either invalid or (4) if a `c_nonce` was provided by the Credential Issuer and one of the key proofs is not linked to a previously issued valid `c_nonce` value (refer to (#issuer-provided-nonce)).
```

Comment:
The "either"s feels odd to me and I think can be omitted without changing the meaning, and the `c_nonce` may not have come from the Credential Issuer so I suggest removing that part:

```suggestion
  * `invalid_proof`: The `proof` or `proofs` parameter in the Credential Request is invalid: (1) if both fields are missing, or (2) both are present simultaneously, or (3) one of the provided key proofs is invalid, or (4) if a `c_nonce` was previously provided and one of the key proofs is not linked to a valid `c_nonce` value (refer to (#issuer-provided-nonce)).
```